all.yml

```diff
@@ -0,0 +1,7 @@
+- hosts: localhost
+  vars:
+    state: present
```

Comment:
Just as an alternative, you can do the ENV lookup here:
```
vars:
  travisCI: {{ lookup('ENV', 'TRAVIS' }}
```

Then you can have `when: travisCI` in the role as opposed to doing env lookup there. Either way looks good!